usr/share/rear/layout/prepare/GNU/Linux/210_load_multipath.sh

```diff
@@ -38,7 +38,10 @@ blacklist {
     multipath >&2
     if [ $? -ne 0 ] ; then
         LogPrint "Failed to activate multipath, or no multipath device found."
-        rear_shell "Did you activate the multipath devices?"
+        user_input="$(UserInput -t 30 -p "Do you need to enter in rear_shell to manually activate multipath ? [type 'yes']")"
+        if [ $user_input == "yes" ]; then
+            rear_shell "Did you activate the multipath devices?"
```

Comment:
Can we somehow check if the user did something useful? What if the user did not really activate multipathing, will the remaining recovery work or will it fail later with an obscure error message?